build.gradle

```diff
@@ -1,7 +1,7 @@
 import java.text.SimpleDateFormat
 
 plugins {
-    id "org.jetbrains.kotlin.jvm" version "1.5.31"
+    id "org.jetbrains.kotlin.jvm" version "1.6.10"
```

Comment:
Had to upgrade Kotest because it didn't seem to want to run the data driven tests in Gradle. The new Kotest required Kotlin `1.6.x` or beyond.